cognite/client/stable/datapoints.py

```diff
@@ -796,13 +796,31 @@ def _get_datapoints_frame_user_defined_limit(self, time_series, aggregates, gran
         return df
 
     def post_datapoints_frame(self, dataframe) -> None:
-        """Write a dataframe
+        """Write a dataframe.
+        The dataframe must have a 'timestamp' column with timestamps in milliseconds.
+        The names of the remaining columns specify the names of the time series to which column contents will be written.
```

Comment:
Maybe also include that the time series to which the data is being posted must already have been created.